### ISSUE TITLE
Verify ArchGym install with test train script example (standalone from vizier install)

### ISSUE THREAD

**Issue by ShvetankPrakash** [srivatsankrishnan/oss-arch-gym#23]
After this section in the [documentation](https://oss-archgym.readthedocs.io/en/latest/installation.html#activate-arch-gym-environment) we should have them run like a working example (i.e. train*.py) to verify install is working before going on to the vizier installation. Also the vizier installation should be on a separate page I think cause the setup page feels kind of cluttered at the moment.

**Comment by ShvetankPrakash** [comment]
@aadyapipersenia04 just tagging you since this is part of what we discussed last Friday

**Comment by TensorTyrion9001** [comment]
@ShvetankPrakash can we use "train_randomwalker.py" to verify arch-gym installation before testing vizier

This file is located in sims/customenv

**Comment by aadyapipersenia04** [comment]
Yes, the documentation has been updated, but it's in the documentation branch, we will merge it with the main branch.

**Comment by ShvetankPrakash** [comment]
Yes this is a good idea @TensorTyrion9001 . Can you and @aadyapipersenia04 make sure this gets updated? @aadyapipersenia04 are you referring to script for testing vizier or script for testing archgym? At the end of this section: https://oss-archgym.readthedocs.io/en/latest/installation.html we should have a script for them to run to verify arch gym installation.
